<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerificarInconsistenciasDialog</class>
 <widget class="QDialog" name="VerificarInconsistenciasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Verificar Inconsistências</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="executarVerificacaoButton">
     <property name="text">
      <string>Executar Verificação</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="resultadosTable">
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tipo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Caminho</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Problema</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="baixarCSVButton">
     <property name="text">
      <string>Baixar Resultados (CSV)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>